### ISSUE TITLE
add a `/ignore <PLAYER>` command 

### ISSUE THREAD

**Issue by fluxionary** [obl3pplifp/BlS-issue-tracker#185]
One player was recently (lightly) harassing another via PMs, and lamented the lack of a way to ignore specific people. I think it's a good idea, but needs the following features:

1. `/ignore [TIMESPAN] <PLAYERNAME>` - stop receiving messages from a given player, w/ an optional timespan (like 1h, 5d etc. - should 1m be "month" or "minute"?)
2. It should work across all chat-based methods of communication (e.g. normal chat and /msg)
3. There should be an easy way to view which players you are currently ignoring
4. There should be an easy way to un-ignore someone. 
5. Staff should be un-ignorable (via a privilege detection or something like that) (thanks for the suggestion, FR)

This could be done w/ an existing mod, a new mod, or changes to bls_custom

**Comment by WulftheNordicRanger** [comment]
Good idea, although staff members should be unignorable for obvious reasons

**Comment by WulftheNordicRanger** [comment]
The forums were having site issues (seems to be a thing for them lately) but I did find this: https://github.com/appgurueu/cmdlib/blob/master/Readme.md (just an api atm but it could be modified

**Comment by oversword** [comment]
Fixed by https://github.com/BlockySurvival/bls_custom/pull/69
Nice.
